angular.json

```diff
@@ -11,7 +11,8 @@
     "schematicCollections": [
       "@cypress/schematic",
       "@schematics/angular"
-    ]
+    ],
+    "analytics": false
```

Comment:
not directly related with the feature I assume 
(next time, just include it in a standalone PR. not to be fixed this time)